### PR TITLE
Actions

### PR DESCRIPTION
**Pull request by jburel**
Replace travis by GHA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-Travis Bionic Zeroc Ice Python 3 Builder
-========================================
+Bionic Zeroc Ice Python 3 Builder
```
Comment (sbesson) [inline README.md]: Comparing with https://github.com/ome/zeroc-ice-ubuntu1804, shoudl we use `Ubuntu 18.04` instead of bionic everywhere?

**Review by sbesson — COMMENTED** [review]
To make sure we agree on the scope esp. in relationship to https://github.com/ome/zeroc-ice-py-manylinux/pull/4#pullrequestreview-551461042, the goal is to support this repository as the source of pre-compiled wheels for all supported Python versions on Ubuntu 18.04 and consume them e.g in GitHub actions. Is that correct?

In that case, should we remove the repository e.g. `zeroc-ice-py-ubuntu1804` to match the semantics of the other Ice repositories?

**Comment by jburel** [comment]
We need to keep one only either that one or zeroc-ice-ubuntu1804. ``zeroc-ice-ubuntu1804`` is used in our installation documentation cf. https://docs.openmicroscopy.org/omero/latest/sysadmins/unix/server-ubuntu1804-ice36.html. So I will favour it and adjust accordingly if required.
Add support for actions in this repo and archived.

**Comment by manics** [comment]
zeroc-ice-py-ubuntu1804 uses the system Python, whereas this repo uses the travis or github VM python distribution which may or may not be identical (maybe someone else knows exactly what determines the compatibility of compiled Python versions with compiled modules?). If you merge the two repos you'll therefore need to verify that both the OS python and the CI python are working.

**Comment by jburel** [comment]
After discussion, we agree
* not to merge the repositories as indicated above
* rename this repository: suggested name ``zeroc-ice-py-github-ci``
* archive https://github.com/ome/zeroc-ice-py-manylinux and redirect to this repository

**Comment by sbesson** [comment]
:+1: merging as per the set of decisions. Are you handling the renaming and README updates as follow-ups or should we capture them as issues?

**Pull request merged**